pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -369,9 +369,9 @@ func (e *etcdObserver) setBootstrapMember() error {
 	for _, subset := range endpoints.Subsets {
 		for _, address := range subset.Addresses {
 			if address.Hostname == "etcd-bootstrap" {
-				name := address.Hostname
-				peerURLs := fmt.Sprintf("https://%s.%s:2380", name, e.ClusterDomain)
-				clusterMember, err := setMember(name, []string{peerURLs}, ceoapi.MemberUnknown)
+				ip := address.IP
```

Comment:
drop the extra variable?

Reply:
it's used more than once I think its fine IMO